### ISSUE TITLE
Allow secret S3 credentials to be specified via environment variables

### ISSUE THREAD

**Issue by lencioni** [Galooshi/happo#91]
Right now, this project requires that the S3 credentials are in your YAML file.

```yaml
s3_access_key_id: <your acccess key id>
s3_secret_access_key: <your secret acccess key>
s3_bucket_name: <a globally unique bucket name>
```

In the example project, [we use Ruby to dynamically read these values in, which is a bit weird](https://github.com/Galooshi/happo/blob/ce49454700aecfede3950298bf85d25ac7b34658/example-project/.happo.yaml#L14-L15).

```yaml
s3_access_key_id: <%= ENV['S3_ACCESS_KEY_ID'] %>
s3_secret_access_key: <%= ENV['S3_SECRET_ACCESS_KEY'] %>
s3_bucket_name: <%= ENV['S3_BUCKET_NAME'] %>
```

Perhaps we should remove these from the configuration file and always require them to be specified via an environment variable?

**Comment by trotzig** [comment]
Sounds good to me.

**Comment by lencioni** [comment]
This was implemented by #92
